contracts/upgradeability/OwnedUpgradeabilityProxy.sol

```diff
@@ -82,4 +92,12 @@ contract OwnedUpgradeabilityProxy is UpgradeabilityProxy {
     upgradeTo(implementation);
     require(this.call.value(msg.value)(data));
   }
+
+  /**
+   * @dev Redefines Proxy's fallback to disallow delegation when caller is the proxy owner.
+   */
+  function () public payable {
+    require(msg.sender != proxyOwner());
```

Comment:
Wouldn't this break the createAndCall methods?

Reply:
I think it shouldn't since the factory performs the call and then transfers the ownership to the requested owner